Resources/Prototypes/_Funkystation/Atmospherics/crystallizer.yml

```
@@ -0,0 +1,309 @@
+  - 0     # frezon
+  - 7.5   # bz
+  - 0     # healium
+  - 0     # nitrium
+  - 0     # pluox
+  - 0     # hydrogen
+  - 0     # hypernob
+  - 0     # protonitrate
+  - 20    # zauker
+  - 0     # halon
+  - 0     # helium
+  - 5     # antinob
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)